test/elpa.el

```diff
@@ -0,0 +1,5 @@
+(setq package-user-dir
+      (expand-file-name (format ".cask/%s/elpa" emacs-version)))
```

Comment:
I am not aware that Cask had any issues in ansi-term nor can I remember any such issues reported.

But then I do not use ansi-term and am not in a position to answer.

If you could report the ansi-term issue to our tracker we might be able to try and fix it.